src/pages/ContactUs.tsx

```diff
@@ -1,10 +1,19 @@
-import icon from '../images/icon.svg'
 import BasicPage from 'src/components/BasicPage'
+import contactEnvelope from '../images/contact-envelope.svg'
+// import contactStripes from '..images/contact-stripes.svg'
```

Comment:
Better to remove the line. leaving comments in is ok if it's necessary to document code, otherwise err on the side of removing.